kpops/components/base_components/kafka_app.py

```diff
@@ -63,33 +67,27 @@ def deploy(self, dry_run: bool):
                 )
         super().deploy(dry_run)
 
-    def destroy(self, dry_run: bool, clean: bool, delete_outputs: bool) -> None:
-        super().destroy(dry_run, clean, delete_outputs)
-
-    def clean(self, dry_run: bool, delete_outputs: bool, clear_schemas: bool):
-        if delete_outputs:
-            # producer clean up job will delete output topics
-            self.clean_app(
-                values=self.to_helm_values(),
-                dry_run=dry_run,
-                delete_outputs=delete_outputs,
-                retain_clean_jobs=self.config.retain_clean_jobs,
-            )
+    @override
+    def destroy(self, dry_run: bool) -> None:
+        super().destroy(dry_run)
 
-            if self.to and self.handlers.schema_handler and clear_schemas:
-                self.handlers.schema_handler.delete_schemas(self.to, dry_run)
+    @override
+    def clean(self, dry_run: bool):
+        self.clean_app(
+            values=self.to_helm_values(),
+            dry_run=dry_run,
+            retain_clean_jobs=self.config.retain_clean_jobs,
+        )
 
     def clean_app(
```

Comment:
if we're exposing the `clean` method as part of the API, this should be internal. you can prefix it using 1 or 2 underscores

Reply:
Good point I did it with 1 underscore to make it protected since the `streams_app.py` is using it